sandstone/lib/handlers/broadcast.py

```diff
@@ -1,19 +1,47 @@
 import tornado.ioloop
 import tornado.web
 import tornado.websocket
+import tornado.escape
 import json
 from pydispatch import dispatcher
 
+
+class BroadcastManager(object):
+    _clients = set()
+
+    @classmethod
+    def add_client(cls, client):
+        cls._clients.add(client)
+
+    @classmethod
+    def broadcast(cls, message):
+        for client in cls._clients:
+            try:
+                client.write_message(message)
+            except:
```

Comment:
Before the final PR, make sure this is replaced with specific exception handling.